backend/services/interfaces/userService.ts

```diff
@@ -1,10 +1,4 @@
-import {
-  CreateUserDTO,
-  Role,
-  SignUpMethod,
-  UpdateUserDTO,
-  UserDTO,
-} from "../../types";
+import { CreateUserDTO, Role, UpdateUserDTO, UserDTO } from "../../types";
```

Comment:
Why was the `SignUpMethod` removed in this PR? What did it do?

Reply:
Oh, I think this changed when running the linter. I can remove it since [this PR ](https://github.com/uwblueprint/rowan-house/pull/48)seems to be handling it.

Reply:
This will also mean that this PR won't pass linter until the other linter PR is merged and so on